src/api-reference/travel-receipts/v4.travel-receipts.markdown

```diff
@@ -0,0 +1,166 @@
+---
+title: Travel Receipts v1
+layout: reference
+---
+
+{% include prerelease.html %}
+
+
+The Travel Receipts service offers one endpoint for retrieving receipt requests.
+
+The Travel Receipts service exposes receipt requests to inform e-receipt partners which e-receipts to send to SAP Concur and for which user.
+
+If the partner supplies a timestamp, it is used as the start time for results. If not, the API retrieves the results from the last 24 hours.
+
+> **Limitations**: This API is not available in Implementation environments. This API is only available in the North American and EMEA Data Centers. This API is only available to SAP Concur e-receipt partners.
+
+* [Products and Editions](#products-editions)
+* [Scope Usage](#scope-usage)
+* [Dependencies](#dependencies)
+* [Access Token Usage](#access-token-usage)
+* [Retrieve a Travel Receipt](#retrieve-travel-receipt-requests)
+* [Schema](#schema)
+
+## <a name="products-editions"></a>Products and Editions
+
+* Concur Travel Professional Edition
+* Concur Travel Standard Edition
+
+## <a name="scope-usage"></a>Scope Usage
+
+Name|Description|Endpoint
+---|---|---
+`travel.receipts.read`|Retrieves a travel receipt.|GET
+
+## <a name="dependencies"></a>Dependencies
+
+None.
+
+## <a name="access-token-usage"></a>Access Token Usage
+
+This API supports company level access tokens.
+
+## <a name="retrieve-travel-receipt-requests"></a>Retrieve Travel Receipt Requests
+
+Retrieves all travel receipt requests for the partner.  
+
+The results of the API call are limited to a maximum of 25 receipt requests per page. The partner can navigate to the next page of results through the `next` field in the response. Each page keeps the same timestamp but will have a different key. The partner has reached the last page of results when `next` is null/empty.
+
+### Scopes
+
+`travel.receipts.read` - Refer to [Scope Usage](#scope-usage) for full details.
+### Request
+
+#### URI
+
+##### Template
+
+```shell
+https://{datacenterURI}/travelreceipts/v1/receiptrequests/{ts}/{key}
+```
+
+##### Parameters
+
+Name | Type | Format | Description
+-----|------|--------|------------
+`ts`	|	`integer`	|	`int64`	|	Timestamp to specify a start time (if blank, defaults to last 24 hours).
```

Comment:
Since this is a net new file we should make sure table formatting conforms to the whitespace conventions in the documentation template. Bonus points for making the prior committed docs do as well.

Reply:
Nice catch. I didn't realize these spaces were actually tabs. Corrected.